script/zlib.decompress.py

```diff
@@ -0,0 +1,20 @@
+import zlib
+import sys
+
+CHUNKSIZE = 1024
+zlib_obj = zlib.decompressobj(int(sys.argv[2]))
+compressed_file = open(sys.argv[1], 'rb')
+buf = compressed_file.read(CHUNKSIZE)
```

Comment:
toto vypada, ze je tu zbytecne (vim, neni), byl by imho lepsi  `while true` https://coderwall.com/p/q_rd1q/emulate-do-while-loop-in-python

Reply:
upravil sem